pkg/session/valkey/repository.go

```diff
@@ -2,15 +2,25 @@ package sessionvalkey
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"time"
 
 	"github.com/valkey-io/valkey-go"
 
 	"github.com/openkcm/session-manager/pkg/session"
 )
 
-const objectTypeSession = "session"
-const objectTypeState = "state"
+const (
+	objectTypeSession         = "session"
```

Comment:
suggestion: I can see many consts prefixed with `ObjectType`. We could create an enum of these if they are interchangeable to easier type picking :)